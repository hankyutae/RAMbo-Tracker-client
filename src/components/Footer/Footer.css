#phantom-footer{
  display: block;
  padding: 0px;
  height: 10vh;
}

#footer{
  display: flex;
  justify-content: space-between;
  align-items: center;
  background: linear-gradient(180deg, rgb(212, 212, 212) 10%, rgb(185, 185, 185) 150%, rgb(212, 212, 212) 10% );
  color: black;
  padding: 15px;
  position: relative;
  left: 0;
  bottom: 0px;
  width: 100%;
  overflow: hidden;
}

#title-footer, .login-footer{
  margin: 0 10px 3px 10px;
}

#copy-footer{
  display:block;
  margin: 0 10px 3px 10px;
  text-align: right;
}

#developers{

text-align: right;
}

#developers li{
  margin:1px;
padding:1px;
}

#app-info{
  text-align: left;
}

#footer a {
  text-decoration: underline;
}
#footer a:hover{
  color: rgb(8, 8, 94);
  text-decoration: underline;
}