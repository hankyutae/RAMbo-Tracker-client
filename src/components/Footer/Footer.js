import React from "react";
import "./Footer.css";

export default function Footer() {
  return (
    <div>
      <div id="footer" role="contentinfo">
        <div id="app-info">
          <div id="title-and-links">
            <p id="title-footer">
              Symptom Tracker<br></br>{" "}
              <a
                href="https://github.com/thinkful-ei-emu/RAMbo-Tracker-client"
                target="_blank"
                rel="noopener noreferrer"
              >
                Client Code
              </a>{" "}
              <br></br>
              <a
                href="https://github.com/thinkful-ei-emu/RAMbo-Tracker-Server"
                target="_blank"
                rel="noopener noreferrer"
              >
                Server Code
              </a>
            </p>
          </div>
          <div id="demo-login">
            <p className="login-footer">Demo Username: testuser</p>
            <p className="login-footer">Demo Password: password1</p>
          </div>
        </div>
        <div id="developers">
          <p id="created-head">Created By:</p>
          <ul>
            <li>
              <a
                href="https://thinkful-ei-emu.github.io/portfolio-colleen/"
                target="_blank"
                rel="noopener noreferrer"
              >
                Colleen Higgins
              </a>
            </li>
            <li>
              <a
                href="https://thinkful-ei-emu.github.io/Paul-Portfolio/"
                target="_blank"
                rel="noopener noreferrer"
              >
                {" "}
                Paul KHan
              </a>
            </li>
            <li>
              <a
                href="https://github.com/Blakelowrey"
                target="_blank"
                rel="noopener noreferrer"
              >
                {" "}
                Blake Lowrey{" "}
              </a>
            </li>
            <li>
              <a
                href="https://thinkful-ei-emu.github.io/portfolio-corey/"
                target="_blank"
                rel="noopener noreferrer"
              >
                Corey Moore
              </a>
            </li>
            <li>
              <a
                href="https://tarajpatel.com"
                target="_blank"
                rel="noopener noreferrer"
              >
                T.J. Patel
              </a>
            </li>
            <li>
              <a
                href="https://dcoollx.github.io/Portfolio/"
                target="_blank"
                rel="noopener noreferrer"
              >
                David Queen Jr.
              </a>
            </li>
          </ul>
          <p id="copy-footer">RAMbo &copy; 2019</p>
        </div>
      </div>
    </div>
  );
}
