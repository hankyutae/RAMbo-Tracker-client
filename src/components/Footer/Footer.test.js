import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter>
      <Footer refreshesWhenAppStateDoes={() => {}} />
    </BrowserRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<BrowserRouter>
      <Footer refreshesWhenAppStateDoes={() => {}} />
    </BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
