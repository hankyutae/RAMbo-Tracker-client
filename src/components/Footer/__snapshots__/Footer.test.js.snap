// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <div
    id="phantom-footer"
  />
  <div
    id="footer"
  >
    <p
      id="title-footer"
    >
      Symptom Tracker
    </p>
    <p
      id="copy-footer"
    >
      RAMbo Â© 2019
    </p>
  </div>
</div>
`;
